exponent_server_sdk/__init__.py

```diff
@@ -399,20 +402,58 @@ def publish_multiple(self, push_messages):
            An array of PushTicket objects which contains the results.
         """
         push_tickets = []
+        session = requests.Session()
+        session.headers.update({
+            'accept': 'application/json',
+            'accept-encoding': 'gzip, deflate',
+            'content-type': 'application/json',
+        })
```

Comment:
A couple of suggestions around this:

1. Unless I'm missing something, I think the session should be created in the `__init__` method and then just used where needed? That would save you recreating the session each time it's called as you do now.
2. It would be more extendible if there were a `get_session()` method for creating a session, or failing that, being able to pass in a session would allow for customisation.

Reply:
1 seems like a good idea.

2. I like the idea of being able to pass in a session, otherwise the default one will be created for you.